src/Sylius/Bundle/CoreBundle/Fixture/Factory/TaxonExampleFactory.php

```diff
@@ -128,9 +151,30 @@ protected function configureOptions(OptionsResolver $resolver): void
             })
             ->setDefault('children', [])
             ->setAllowedTypes('children', ['array'])
+            ->setDefault('images', [])
+            ->setAllowedTypes('images', ['array'])
         ;
     }
 
+    /**
+     * @param array $image
+     * @return TaxonImageInterface
+     */
+    private function createImages(array $image): TaxonImageInterface
```

Comment:
`['foo.jpg', 'this', 'does', 'not', 'matter', 'thumbnail']` has the same effect as `['foo.jpg', 'thumbnail']`